docs/en/observability/apm/collect-application-data/aws-lambda-extension.asciidoc

```diff
@@ -1,6 +1,15 @@
+////
+TO DO:
+Does this need to be its own page?
+////
+
```

Comment:
I think it's useful to see that this is a feature we support in the TOC. I'd be curious to hear if @AlexanderWert still thinks this is important to have [a dedicated page](https://observability-docs_bk_4230.docs-preview.app.elstc.co/guide/en/observability/master/apm-monitoring-aws-lambda.html) or if it's okay for this to just be a sub-heading on [this page](https://observability-docs_bk_4230.docs-preview.app.elstc.co/guide/en/observability/master/apm-collect-application-data.html).

Reply:
Let's keep this page for now, and we can revisit this question in https://github.com/elastic/obs-docs-projects/issues/269.